### PR TITLE
Block reward

### PR DESCRIPTION
**Pull request by jiwon-lieb**
## Proposed changes

- Describe your changes to communicate to the maintainers why we should accept this pull request.
- If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Minor Issues and Typos
- [ ] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn-docs/blob/master/CONTRIBUTING.md)
- [ ] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn-docs)
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large content contribution, kick off the discussion by explaining why you would suggest the content contribution, etc...

### CODE REVIEW

File: docs/dapp/json-rpc/api-references/governance.md [hunk 1]
```diff
@@ -457,7 +457,7 @@ The `itemCacheFromDb` returns the governance information stored in the given blo
   reward.minimumstake: "5000000",
   reward.mintingamount: "9600000000000000000",
```
Comment (dcground) [inline docs/dapp/json-rpc/api-references/governance.md:458]: 96 --> 64

File: docs/dapp/sdk/caver-js/api-references/caver.rpc/governance.md [hunk 2]
```diff
@@ -201,7 +201,7 @@ Provides the initial chain configuration. Because it just stores the initial con
     governanceMode: 'single',
     reward: {
       mintingAmount: 9600000000000000000,
```
Comment (dcground) [inline docs/dapp/sdk/caver-js/api-references/caver.rpc/governance.md:203]: 96 --> 64

File: docs/dapp/sdk/caver-js/api-references/caver.rpc/governance.md [hunk 3]
```diff
@@ -291,7 +291,7 @@ Returns governance items at a specific block. It is the result of previous votin
   'reward.minimumstake': '5000000',
   'reward.mintingamount': '9600000000000000000',
```
Comment (dcground) [inline docs/dapp/sdk/caver-js/api-references/caver.rpc/governance.md:292]: 96-->64

File: docs/dapp/sdk/caver-js/api-references/caver.rpc/governance.md [hunk 4]
```diff
@@ -434,7 +434,7 @@ Returns the governance information stored on the given block. If no changes are
...
```
Comment (dcground) [inline docs/dapp/sdk/caver-js/api-references/caver.rpc/governance.md:435]: 96 --> 64

File: docs/installation-guide/deployment/service-chain/references/genesis.md [hunk 5]
```diff
@@ -126,7 +126,7 @@ The `reward` field stores the information about the network's token economy.
             "governanceMode": "single",
             "reward": {
                 "mintingAmount": 9600000000000000000,
```
Comment (dcground) [inline docs/installation-guide/deployment/service-chain/references/genesis.md:128]: 96 -->64

File: docs/dapp/json-rpc/api-references/governance.md [hunk 6]
```diff
@@ -255,7 +255,7 @@ None
       minimumStake: 5000000,
       mintingAmount: 9600000000000000000,
```
Comment (jiwon-lieb) [inline docs/dapp/json-rpc/api-references/governance.md:256]:
```suggestion
      mintingAmount: 6400000000000000000,
```

File: docs/dapp/json-rpc/api-references/governance.md (hunk 1 above)
Comment (jiwon-lieb) [inline docs/dapp/json-rpc/api-references/governance.md:458]:
```suggestion
  reward.mintingamount: "6400000000000000000",
```

File: docs/dapp/sdk/caver-js/api-references/caver.rpc/governance.md (hunk 2 above)
Comment (jiwon-lieb) [inline docs/dapp/sdk/caver-js/api-references/caver.rpc/governance.md:203]:
```suggestion
      mintingAmount: 6400000000000000000,
```

File: docs/dapp/sdk/caver-js/api-references/caver.rpc/governance.md (hunk 3 above)
Comment (jiwon-lieb) [inline docs/dapp/sdk/caver-js/api-references/caver.rpc/governance.md:292]:
```suggestion
  'reward.mintingamount': '6400000000000000000',
```

File: docs/dapp/sdk/caver-js/api-references/caver.rpc/governance.md (hunk 4 above)
Comment (jiwon-lieb) [inline docs/dapp/sdk/caver-js/api-references/caver.rpc/governance.md:435]:
```suggestion
  'reward.mintingamount': '6400000000000000000',
```

File: docs/installation-guide/deployment/service-chain/references/genesis.md (hunk 5 above)
Comment (jiwon-lieb) [inline docs/installation-guide/deployment/service-chain/references/genesis.md:128]:
```suggestion
                "mintingAmount": 6400000000000000000,
```

**Review by praveen-klaytn — APPROVED** [review]

**Review by kjhman21 — APPROVED** [review]

**Pull request merged**